common/app/views/main.scala.html

```diff
@@ -27,7 +27,7 @@
 }
 
 <!DOCTYPE html>
-<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+<html id="js-context" class="js-off is-not-modern id--signed-out" lang="en" data-page-path="@request.path">
```

Comment:
Personally I'd contend that since *only* the article copy is in a foreign language, and everything surrounding it is english (navigation, header, etc), it's not like we're really making a concerted effort or accepting responsibility for the site to be navigable and accessible to non-english users, we just happen to have some non-english text there, so "en" is more appropriate.